src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -466,6 +472,42 @@ private void mpathwaysCall(HttpServletRequest request, HttpServletResponse respo
 		out.flush();
 	}
 
+	public void addMpathwayDataToSession(HttpServletRequest request,
```

Comment:
Is this adding all Mpathways data or just termid / classnumber?  Just don't want someone to come along later and assume this is general purpose if it isn't.